### PR TITLE
Adds: PID cut configurables in the PID task

### PR DESCRIPTION
**Pull request by sbhawani**

### CODE REVIEW

File: Detectors/TPC/qc/include/TPCQC/PID.h [hunk 1]
```diff
@@ -68,27 +64,32 @@ class PID
   /// Dump results to a file
   void dumpToFile(std::string filename);
 
+  // To set the elementary track cuts
+  void setPIDCuts(int nClusterCut = 60, float AbsTgl = 1., float MindEdxTot = 10.0,
+                  float MaxdEdxTot = 70., float MinpTPC = 0.05, float MaxpTPC = 20., float MinpTPCMIPs = 0.45, float MaxpTPCMIPs = 0.55)
```
Comment (wiechula) [inline Detectors/TPC/qc/include/TPCQC/PID.h]: Naming:
```suggestion
  void setPIDCuts(int minnCls = 60, float absTgl = 1., float mindEdxTot = 10.0,
                  float maxdEdxTot = 70., float minpTPC = 0.05, float maxpTPC = 20., float minpTPCMIPs = 0.45, float maxpTPCMIPs = 0.55)
```

**Review by wiechula — CHANGES_REQUESTED** [review]

File: Detectors/TPC/qc/include/TPCQC/PID.h (hunk 1 above)
Comment (sbhawani) [reply Detectors/TPC/qc/include/TPCQC/PID.h]: Changed them now

**Comment by sbhawani** [comment]
@wiechula @davidrohr and @shahor02, this PR is ready to be merged!, thanks

**Review by wiechula — APPROVED** [review]

**Comment by sbhawani** [comment]
This error seems not related to  codes change in this PR apparently

**Comment by tklemenz** [comment]
Yes, the error is unrelated. Can it be merged?

**Comment by wiechula** [comment]
@davidrohr , @shahor02 , errors look unrelated. Can this be merged?

**Pull request merged**